src/main/kotlin/com/vk/admstorm/ssh/EnterPasswordDialog.kt

```diff
@@ -2,14 +2,18 @@ package com.vk.admstorm.ssh
 
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.ui.JBColor
 import com.intellij.ui.components.JBCheckBox
 import com.intellij.ui.components.JBPasswordField
 import com.intellij.ui.dsl.builder.AlignX
 import com.intellij.ui.dsl.builder.BottomGap
 import com.intellij.ui.dsl.builder.TopGap
 import com.intellij.ui.dsl.builder.panel
 import com.intellij.util.ui.JBDimension
+import java.awt.event.KeyAdapter
+import java.awt.event.KeyEvent
 import javax.swing.JComponent
+import javax.swing.JLabel
 import javax.swing.JPanel
 
 class EnterPasswordDialog(project: Project) : DialogWrapper(project, true, IdeModalityType.PROJECT) {
```

Comment:
you might as well get rid of the `my` prefix

Reply:
Done